dvc/stage/__init__.py

```diff
@@ -215,9 +225,9 @@ def _changed_deps(self):
             status = dep.status()
             if status:
                 logger.debug(
-                    "Dependency '{dep}' of '{stage}' changed because it is "
+                    "Dependency '{dep}' of {stage} changed because it is "
```

Comment:
"of stage: 'mystage'" looks very odd, especially that `:`

Reply:
I do like it though:
```
Dependency 'dep' of stage: 'stage.dvc' changed because it is
```

I don't think stage needed to be uppercased.

Reply:
yeah, but it doesn't need `:` too, right?

Reply:
Yes, it does not need to there. I'll remove it.

Reply:
It looks kinda odd in `str(stage)`, but, I can just reserve it for presentational purpose. Removing ...